composer.json

```diff
@@ -35,6 +35,7 @@
         "php": "^7.3 || ^8",
         "ext-pdo": "*",
         "doctrine/cache": "^1.0",
+        "doctrine/deprecations": ">=0.3.0",
```

Comment:
Why is the change from `^0.2` to this necessary? Is `doctrine/deprecations` expected to be backward-compatible forever?

Reply:
My bad, i was expecting `^0.2.0` to also install `^0.3.0` but during  the `0.` branch the caret is actually more strict and `^0.2.0` means `>=0.2.0,<0.3.0` and not `<1.0.0`. I guess that means as long as deprecations doesn't break BC new features should get into mini versions for now, or we should release `1.0` very quickly.

Reply:
With the major version being `0`, the minor part is promoted to major. Probably the same applies to the patch promoted to being the minor. I think it's okay to introduce features in patches before a stable version is released because ([ref](https://semver.org/spec/v0.1.0.html#spec-item-5)):
> Major version zero (0.y.z) is for initial development. Anything may change at any time. The public API should not be considered stable.